schema.sql

```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals (
+    id INT,
```

Comment:
[OPTIONAL]
- I would suggest you set the `id` column as the primary key. Columns to be set as primary keys are those that contain unique values and this makes it easy to identify each record/row on the table. 😉 
Kindly go through this [W3Schools article](https://www.w3schools.com/sql/sql_primarykey.ASP) to learn more.

- Another suggestion that I would give would be for the `id` to be set as `Not Null` so that it is never empty. This will help to ensure that a value is present for the `id` column before the record being added is saved/committed. You can also set the `id` column as `GENERATED ALWAYS AS IDENTITY`. This ensures that sequential integers that cannot be altered are generated each time a record is being created. I hope it makes sense. 😄 
You can go through this [article](https://www.sqltutorial.org/sql-identity/#:~:text=The%20GENERATED%20ALWAYS%20generates%20sequential,integers%20for%20the%20identity%20column.).